purchases-sample/src/main/java/com/revenuecat/sample/OfferingFragment.kt

```diff
@@ -48,16 +60,16 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
             currentPackage,
             { error, userCancelled ->
                 if (!userCancelled) {
-                    showError(error)
+                    showUserError(requireActivity(), error)
                 }
             }) { purchaseDetails, _ ->
-                Toast.makeText(
-                    requireContext(),
-                    "Successful purchase, order id: ${purchaseDetails.orderId}",
-                    Toast.LENGTH_LONG
-                ).show()
+            Toast.makeText(
+                requireContext(),
+                "Successful purchase, order id: ${purchaseDetails.orderId}",
+                Toast.LENGTH_LONG
+            ).show()
 
-                findNavController().navigateUp()
+            findNavController().navigateUp()
```

Comment:
I think I would have the onSuccess lambda as another parameter so this indentation doesn't look weird:

```
      Purchases.sharedInstance.purchasePackageWith(
            requireActivity(),
            currentPackage,
            { error, userCancelled ->
                if (!userCancelled) {
                    showUserError(requireActivity(), error)
                }
            },
            { purchaseDetails, _ ->
                Toast.makeText(
                    requireContext(),
                    "Successful purchase, order id: ${purchaseDetails.orderId}",
                    Toast.LENGTH_LONG
                ).show()

                findNavController().navigateUp()
            })
            ```